docs/src/componentDocs/GettingStarted/ReactNativeTesting.mdx

```diff
@@ -0,0 +1,146 @@
+
+## Testing Frameworks
+
+React applications scaffolded with Create React App, or CRA, was the official way to start a new React project since its introduction in 2016. Unfortunately, CRA was deprecated in 2023. React applications scaffolded by create-react-app (CRA) came with Jest, a built-in unit testing framework.
+
+The Brightlayer UI team researched modern ways to scaffold new React projects with the Brightlayer UI [React Component Library](https://github.com/etn-ccis/blui-react-component-library). Today and moving forward the Brightlayer UI
+team adopts [Vite](https://v3.vitejs.dev/guide/why.html), a platform-agnostic frontend tool for building web applications. When using the Brightlayer UI [React Vite templates](https://github.com/etn-ccis/blui-react-cli-templates) to scaffold your new React project it will include [Vitest](https://vitest.dev/guide/why.html), the unit testing framework for React Vite projects.
+
+Vitest offers compatibility with most of the Jest API and ecosystem libraries and transitioning to Vitest API is highly compatible with Jests API, so existing Jest knowledge can be applied.
+
+If your team prefers Jest, it is possible to use Jest in Vite React setups. [Vite-jest](https://www.npmjs.com/package/vite-jest) integration for Vite and Jest, so this is a valid option for your unit tests but the preference for Brightlayer UI React Vite applications will be to use Vitest testing framework.
+
+### React Vite
+
+The Brightlayer UI [React Vite template](https://github.com/etn-ccis/blui-react-cli-templates) projects are pre-configured with [Vitest](https://v3.vitejs.dev/guide/) testing framework. When creating a new React Vite project from Brightlayer UI templates, a sample test is created in your project for you (test files are identified by the `.spec.tsx` or `.test.tsx` file extension). You can execute the tests by running the following in your terminal:
+
+<CodeBlock code={`npm run test`} language={'sh'} />
+or
+<CodeBlock code={`yarn test`} language={'sh'} />
+
+This will run your tests and show the pass/fail output in the terminal.
+
+There are additional utilities that complement Vitest well. See the [Vitest Environment](https://vitest.dev/guide/environment.html#test-environment) and [features](https://vitest.dev/guide/features.html).
+For testing React components in a Vite project using Vitest, you'll primarily use the following libraries:
+
+1. [Vitest](https://v3.vitejs.dev/guide/): The testing framework itself, which is a fast alternative to Jest, designed specifically for Vite projects.
+2. [React Testing Library](https://github.com/testing-library/react-testing-library#readme): A library that encourages testing your components from the user's perspective, interacting with them as a real user would.
+3. [Jest-dom](https://github.com/testing-library/jest-dom#readme): A library providing custom matchers for testing DOM elements, making your assertions more readable and maintainable.
+
+# End-to-End Testing
+
```

Comment:
We never recommended any E2E testing tools before for react-native given it always had a cost value and nothing was really free for device testing. We might need to skip this section and reach out to adopters and see what test tools are being used and if it requires license per seat deal

Reply:
Removed the testing section from drawer and kept the testing section under the react native section.